yt/go/bus/bus.go

```diff
@@ -13,6 +14,9 @@ import (
 	"go.ytsaurus.tech/library/go/core/log/nop"
 	"go.ytsaurus.tech/yt/go/crc64"
 	"go.ytsaurus.tech/yt/go/guid"
+	"go.ytsaurus.tech/yt/go/proto/core/misc"
+	"go.ytsaurus.tech/yt/yt_proto/yt/core/bus/proto"
+	"google.golang.org/protobuf/proto"
```

Comment:
In go imports usually have 3 groups:
```
import (
    stdlib // os, fmt, context ...
    
    external libs // github.com ...
    
    local libs // go.ytsaurus.tech
)
```

Groups are separated by empty line. In each group imports are sorted.

Most IDE formatting extensions support `-local` flag for correct sorting.

Reply:
Fixed